src/FreeRTOS/FreeRTOSConfig.h

```diff
@@ -65,6 +65,11 @@
 #define configUSE_APPLICATION_TASK_TAG          0
 #define configUSE_COUNTING_SEMAPHORES           1
 
+#define configAPPLICATION_ALLOCATED_HEAP        1   

```

Comment:
maybe to `#ifndef`?

Reply:
I mean - use this define only if user didn't set it himself

Reply:
fixed in new commit